bot/cogs/weather_cog.py

```diff
@@ -0,0 +1,206 @@
+        page += f'Location: {city} ({Lat},{Lon})\n'
+
+        # Current Temperature
+        temp = res_json.get('current',{}).get('temp',{})
+        page += f'Temperature: {temp}°F\n'
+
+        # Weather Conditions
+        desc = res_json.get('current',{}).get('weather',{})[0].get('description',{}).title()
+        page += f'Condition:\t{desc}\n\n'
+        
+        # Feels Like
+        feels = res_json.get('current',{}).get('feels_like',{})
+        page += f'Feels Like: {feels}°F\n'
+
+        # Humidity
+        hum = res_json.get('current',{}).get('humidity',{})
+        page += f'Humidity: {hum}%\n'
+
+        # Wind Speed and Direction
+        wind = res_json.get('current',{}).get('wind_speed',{})
+        wind_deg = res_json.get('current',{}).get('wind_deg',{})
+
+        # Convert Wind Degrees to Direction
+        # Per http://snowfence.umn.edu/Components/winddirectionanddegrees.htm
+        dirs = ['N', 'NNE', 'NE', 'ENE', 'E', 'ESE', 'SE', 'SSE', 'S', 'SSW', 'SW', 'WSW', 'W', 'WNW', 'NW', 'NNW']
+        ix = round(float(wind_deg) / (360. / len(dirs)))
+        wind_dir = dirs[ix % len(dirs)]
+
+        page += f'Wind: {wind} mph ({wind_dir})'
+        
+        pages.append(page)
+        
+        # MULTI-PAGE: Forecast
+        for i in range(8):
+            page=''
+            
+            # Temperatures
+            day_temp = res_json.get('daily',{})[i].get('temp',{}).get('day',{})
+            night_temp = res_json.get('daily',{})[i].get('temp',{}).get('night',{})
+            
+            # Chance of Precipitation
+            precp = res_json.get('daily',{})[i].get('pop',{})
+            
+            # Weather Conditions
+            desc = res_json.get('daily',{})[i].get('weather',{})[0].get('description',{}).title()
+
+            # Humidity
+            hum = res_json.get('daily',{})[i].get('humidity',{})
+
+            # Wind Speed and Direction
+            wind = res_json.get('daily',{})[i].get('wind_speed',{})
+            wind_deg = res_json.get('daily',{})[i].get('wind_deg',{})
+            ix = round(float(wind_deg) / (360. / len(dirs)))
+            wind_dir = dirs[ix % len(dirs)]
+
+            if i > 1:
+                page += f'Day {i}: {day_temp}°F\n'
+                page += f'Night {i}: {night_temp}°F\n'
+            elif i == 1:
+                page += f'Tomorrow: {day_temp}°F\n'
+                page += f'Tomorrow Night: {night_temp}°F\n'
+            else:
+                page += f'Today: {day_temp}°F\n'
+                page += f'Tonight: {night_temp}°F\n'
+            
+            page += f'Condition: {desc}\n\n'
+            page += f'Chance of Precipitation: {round(precp*100)}%\n'
+            page += f'Humidity: {hum}%\n'
+            page += f'Wind: {wind} mph ({wind_dir})\n\n'
+            pages.append(page)
+
+            '''
+            if i % 2 != 0 and i != 7:
+                pages.append(page)
+                page = ''
+            else:
```

Comment:
do we need this code?

Reply:
Nope; since each day takes up enough space to fill a "page" it isn't needed.

Reply:
then it can be deleted